draft-tuexen-opsawg-pcapng.xml

```diff
@@ -1371,6 +1400,39 @@ Section Header
 
             <t>Example: '0'.</t>
 
+            <t hangText="epb_packetid:"><vspace blankLines="0"/>The
+            epb_packetid option is a 64-bit unsigned integer that uniquely
+            identifies the packet. If the same packet is seen by multiple
+            interfaces and there is a way for the capture application to
+            correlated them, the same epb_packetid value must be used. An
+            example could be where an XDP program on ingress captures a packet,
+            which gets modified and then exits the XDP program. In this case,
+            two packets are in the capture file, which are not identical but
+            the epb_packetid can be used correlate them.</t>
```

Comment:
So the capture has two packets that are different from each other but that are the same?

This probably needs to be rephrased in a way to make it clearer that "the packet" doesn't refer to the exact sequence of octets.  Perhaps it should include a less-exotic example than an XDP one, such as a packet captured on a multiple-interface router that arrives on one interface, gets its IP TTL reduced, and is transmitted on another interface.

Reply:
Like the idea, what about changing the example to:

An example could be a router that captures packets on all its interfaces in both directions. When a packet hits interface A on ingress, an ECB entry gets created, TTL gets decremented, and right before it egresses on interface B another ECB entry gets created in the trace file. In this case, two packets are in the capture file, which are not identical but the epb_packetid can be used to correlate them.

Reply:
What's an "ECB"?

Reply:
Ooops, should be EPB :(